kmk/modules/modtap.py

```diff
@@ -21,6 +22,17 @@ def before_matrix_scan(self, keyboard):
         return
 
     def after_matrix_scan(self, keyboard):
+        '''Before other key down decide to send tap kc down.'''
+        if self.matrix_detected_press(keyboard):
+            for key, state in self.key_states.items():
+                if not state.activated:
+                    # press tap because interrupted by other key
+                    self.key_states[key].activated = 'interrupt'
```

Comment:
again preference to making these consts or part of an Enum-esque class, and not magic strings